k8s/webhooks.go

```diff
@@ -271,49 +272,16 @@ func (w *WebhookServer) HandleMutateHTTP(writer http.ResponseWriter, req *http.R
 	}
 
 	// Run the controller
-	mResp, err := controller.Mutate(admReq)
+	mResp, err := controller.Mutate(context.Background(), admReq)
```

Comment:
Same here – we should use the real context:
```suggestion
	mResp, err := controller.Mutate(req.Context(), admReq)
```